com.apifox.apifox.json

```diff
@@ -0,0 +1,165 @@
+                "install -Dm755 run.sh /app/bin/run.sh",
+                "install -Dm755 apply_extra ${FLATPAK_DEST}/bin/apply_extra"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "com.apifox.apifox.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "com.apifox.apifox.metainfo.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "com.apifox.apifox.svg"
+                },
+                {
+                    "type": "file",
+                    "path": "run.sh"
+                },
+                {
+                    "type": "extra-data",
+                    "filename": "apifox_linux.tar.xz",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://file-assets.apifox.com/download/Apifox-linux-manual-latest.tar.gz",
+                    "sha256": "792595282eab3fb2c2902d12b0495cc16cc3c65c763983eead49c0b6645beec4",
+                    "size": 272837328,
+                    "x-checker-data": {
+                        "type": "rotating-url",
+                        "url": "https://file-assets.apifox.com/download/Apifox-linux-manual-latest.tar.gz"
+                    }
+                },
+                {
+                    "type": "extra-data",
+                    "filename": "apifox_linux.zip",
+                    "only-arches": [
+                        "aarch64"
+                    ],
+                    "url": "https://file-assets.apifox.com/download/Apifox-linux-arm64-latest.zip",
+                    "sha256": "4b64237e4455a0d4d4f4ef7ba5cd764baf317c80eb74969e85806d8b779428e9",
+                    "size": 257883364,
+                    "x-checker-data": {
+                        "type": "rotating-url",
+                        "url": "https://file-assets.apifox.com/download/Apifox-linux-arm64-latest.zip"
+                    }
+                },
+                {
+                    "type": "script",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "commands": [
+                        "tar xf apifox_linux.tar.xz",
+                        "rm -rf apifox_linux.tar.xz",
+                        "mv ./*/* ."
+                    ],
+                    "dest-filename": "apply_extra"
+                },
+                {
+                    "type": "script",
+                    "only-arches": [
+                        "aarch64"
+                    ],
+                    "commands": [
+                        "unzip apifox_linux.zip",
+                        "rm -rf apifox_linux.zip",
+                        "mv ./*/* ."
+                    ],
+                    "dest-filename": "apply_extra"
```

Comment:
no need to duplicate the apply_extra? Commands are the same for both arches.

Reply:
ah why aren't you using `tar` here too?

Reply:
is `.zip` file, can  do that?

Reply:
Ah I didn't notice the extension is different.